doc/rfc-move-sigverify-out-of-mask-rom.md

```diff
@@ -0,0 +1,80 @@
+# RFC: Move sigverify out of the mask ROM
+
+Author: James Wainwright
+Date: 2025-04-17
+Status: draft
+
+## Summary
+
+This RFC proposes moving the silicon creator signature verification code from the mask ROM to the immutable ROM extension, which would no longer be optional nor signed.
+
+The mutable ROM extension would be renamed to the FMC (first mutable code), and the term ROM extension would refer only to what we currently call the immutable ROM extension.
```

Comment:
Because of the confusion that's been created with the term "immutable ROM extension", we stopped using that term.  We now use the term "immutable section".

Reply:
Thanks, decided to drop this part of the proposal